libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/PasscodeActivity.java

```diff
@@ -203,7 +203,7 @@ public void enableLogout(boolean b) {
     public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
         Log.i("onEditorAction", "view=" + v + " actionId=" + actionId + " event=" + event);
         // Processing the editor action only on key up to avoid sending events like pass code manager unlock twice.
-        if (event != null && event.getAction() == KeyEvent.ACTION_UP) {
```

Comment:
so after calling setText, call doEditorAction with EditorInfo.IME_ACTION_GO, the event should be set? While with the debug, after  doEditorAction, the even is null, seems like not enter Enter manually after type a passcode there.